### DISCUSSION TITLE
PCB Component Detection

### DISCUSSION THREAD

**Discussion by s39674** [s39674/Image2schematic#21] [Announcements]
The final schematic should include resistors, capacitors, inductors etc.. So we need a way to detect those component.
There is a nice dataset of PCB components here: https://sites.google.com/view/chiawen-kuo/home/pcb-component-detection but I don't know how to start training a model, can anyone help? Thanks!

**Comment by s39674** [comment]
UPDATE:
I created https://github.com/s39674/PCB-Component-Detection with very beginner model similar to this example: https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#sphx-glr-beginner-blitz-cifar10-tutorial-py ; I'm looking for feedback and suggestions.

**Reply by DeepnCode** [reply]
Hii, I am working on a similar project but the basic foundation is the same, what I am doing is to recognize all the pcb components and then generate a BOM list. How can I contact you? Perhaps an email would work, looking forward to your reply! We can discuss the progress and main issues and might work together to help each other out.
